templates/debug.php

```diff
@@ -888,3 +934,32 @@ class="dashicons dashicons-download"></i> <?php fs_esc_html_echo_inline( 'Downlo
         });
     </script>
 <?php endif ?>
+<script type="text/javascript">
+    // JavaScript to toggle the visibility of the table body and change the caret icon
+    document.addEventListener('DOMContentLoaded', function () {
+        document.querySelectorAll('.toggle-table').forEach(function (button) {

```

Comment:
While the codebase still contains some jQuery usage, I believe it is beneficial for us to reduce dependency on it over time, as it’s likely to be phased out in the future.

For this specific block, I opted not to use jQuery because it is entirely new and independent from the other scripts on the page. Additionally, the functionality is simple enough to implement without relying on the library.